packages/components/organisms/f-checkout/src/services/analytics.js

```diff
@@ -0,0 +1,84 @@
+import Trak from '@justeat/f-trak';
+
+const fieldMapper = {
```

Comment:
Would be good to add a comment as to the format of this. I'm guessing it maps names (on the left) to GA name values (on the right), but that's just me assuming so would be good to clarify in a comment